nexus-config/src/nexus_config.rs

```diff
@@ -1049,6 +1052,9 @@ mod test {
             [deployment.dropshot_internal]
             bind_address = "10.1.2.3:4568"
             default_request_body_max_bytes = 1024
+            [deployment.dropshot_debug]
+            bind_address = "10.1.2.3:4569"
```

Comment:
This allows a full socketaddr, but the DNS helper function only allows supplying a debug _port_. Is the expectation that the IP for this must match `dropshot_internal`'s? (Do we check that at runtime? Still reading the PR...)